src/Workspaces/CoreTestUtilities/OptionsCollection.cs

```diff
@@ -64,9 +66,9 @@ public void Add<T>(PerLanguageOption2<CodeStyleOption2<T>> option, T value, Noti
         public void Add(OptionsCollection? options)
             => AddRange(options);
 
-        public void AddRange(OptionsCollection? options)
+        public void AddRange(IEnumerable<KeyValuePair<OptionKey2, object?>>? options)
         {
-            if (options is null)
+            if (options == null)
```

Comment:
nit.  why?

Reply:
Hmm, not sure.

Reply:
I think I removed the check and then added it back or something...